internal/proxy/options_test.go

```diff
@@ -135,6 +152,12 @@ func TestProviderURLValidation(t *testing.T) {
 					t.Errorf("expected SignInURL = %q, got %q", tc.expectedSignInURL, p.SignInURL.String())
 				}
 			}
+			if tc.expectedProxyProviderURLString != "" {
```

Comment:
this is equivalent to `testutil.Equal(t, o.provider.Data().ProxyProviderURL.String(), tc.expectedProxyProviderURLString)`

Reply:
switched to use `testutil.Equal`